nixos/modules/services/misc/ankisyncd.nix

```diff
@@ -0,0 +1,116 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ankisyncd;
+
+  defaultUser = "ankisyncd";
+
+  authDbPath = "${cfg.dataDir}/auth.db";
+
+  sessionDbPath = "${cfg.dataDir}/session.db";
+
+  configFile = pkgs.writeText "ankisyncd.conf" (lib.generators.toINI {} {
+    sync_app = {
+      host = cfg.host;
+      port = cfg.port;
+      data_root = cfg.dataDir;
+      auth_db_path = authDbPath;
+      session_db_path = sessionDbPath;
+
+      base_url = "/sync/";
+      base_media_url = "/msync/";
+    };
+  });
+in
+  {
+    options.services.ankisyncd = {
+      enable = mkEnableOption "ankisyncd";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.ankisyncd;
+        defaultText = literalExample "pkgs.ankisyncd";
+        example = literalExample "pkgs.ankisyncd";
```

Comment:
Nit: having both `defaultText` and `example` being the same value doesn't bring much, so this line could probably be deleted.

Reply:
Good point, fixed it